clients/usageMonitor.lua

```diff
@@ -6,15 +6,41 @@ if not settings.get("misc.monitor") then
   settings.set("misc.monitor", monitorSide)
   settings.save()
 end
+if settings.get("misc.wireless") == nil then
+  settings.define("misc.wireless",{ description = "Should this monitor be in wireless mode? (Use websocket)", type = "boolean"})
+  settings.define("misc.websocketURL",{ description = "URL of the websocket to use for wireless communication", type = "string" })
+  print("Should this operate in wireless mode?")
+  print("Wireless mode would be for connecting without a modem.")
+  print("Otherwise, you will need to connect to a modem on the MISC network.")
+  print("y/n? ")
+  local choice
+  while choice ~= 'y' and choice ~= 'n' do
+      choice = read()
+  end
+  wirelessMode = choice == 'y'
+  settings.set("misc.wireless", wirelessMode)
+  if wirelessMode then
+      print("Enter the URL of the websocket relay service you would like to use.")
+      settings.set("misc.websocketURL", read())
+  end
+  settings.save()
+end
```

Comment:
`wirelessMode` is a global, and never used outside of this situation. This manual setup can also be swapped out for some basic form of automatic detection, so users setting it up in wired mode won't be bothered by this.
```lua
local wirelessMode = fs.exists("websocketLib.lua")
if wirelessMode and not settings.get("misc.websocketURL") then
  settings.define("misc.websocketURL",{ description = "URL of the websocket to use for wireless communication", type = "string" })
  print("Enter the URL of the websocket relay service you would like to use.")
  settings.set("misc.websocketURL", read())
  settings.save()
end
```